.github/workflows/run-target-test-suite.yml

```diff
@@ -0,0 +1,97 @@
+# This workflow runs a specific regression test suite, using the input image
```

Comment:
I think the naming of this workflow is my fault, but if we're assuming that test suites might not all by Jupyter notebook based (e.g., some might use `pytest` or other test frameworks), then this workflow and file should likely be renamed to call out "Jupyter" or "notebook" or something similar.